1_fetch/src/get_wqp_inventory.R

```diff
@@ -0,0 +1,46 @@
+#' Inventory WQP within area of interest
+#' 
+#' @description Function to inventory WQP sites and observations within the area of interest
+#' 
+#' @param bbox_tbl data frame containing an attribute id and bounding box for each
+#' box sampled within the area of interest. The column bbox is of class list.
+#' @param char_names character string indicating the WQP CharacteristicNames to query
+#' @param site_type character string indicating which water feature type(s) to query
+#' @param year_earliest integer; the earliest year to include in the returned inventory,
+#' defaults to NULL, which returns all available years
+#' @param year_latest integer; the most recent year to include in the returned inventory,
+#' defaults to NULL, which returns all available years
+#' 
+#' @value returns a list of sites from the Water Quality Portal
+#' 
+#' @example inventory_wqp(aoi_poly, "Conductivity", "Stream")
+#' @example inventory_wqp(aoi_poly, "Temperature", "Lake, Reservoir, Impoundment")
+#' 
+
+inventory_wqp <- function(bbox_tbl, char_names, site_type, year_earliest = NULL, year_latest = NULL){
+  
+  # Return bounding box to use for WQP data summary
+  bbox <- bbox_tbl$bbox[[1]]
```

Comment:
If you decide to move forward with the simpler bbox mapping approach suggested above, this would not be needed and the incoming arg could just be called `bbox`